learning/tour-of-beam/learning-content/final-challenge/final-challenge-1/description.md

```diff
@@ -0,0 +1,25 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+### Final challenge 1
+
+You’re given a csv file with purchase transactions. Write a Beam pipeline to prepare a send report every 30 seconds. The report needs to be created only for transactions where quantity is more than 20.
+
+Report should consist of two files named "**price more than 10**" and "**price less than 10**":
+
+* Total transactions amount grouped by **ProductNo** for products with **price** greater than 10
+* Total transactions amount grouped by **ProductNo** for products with **price** less than 10
+
+| TransactionNo | Date      | ProductNo | ProductName                         | Price | Quantity | CustomerNo | Country        |
```

Comment:
Could we add a description like: Example rows from input file

Reply:
Done